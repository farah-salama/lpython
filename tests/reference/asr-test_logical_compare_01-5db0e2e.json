{
    "basename": "asr-test_logical_compare_01-5db0e2e",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/test_logical_compare_01.py",
    "infile_hash": "467dc216d8ce90f4b3a1ec06610cea226ae96152763cfa42d5ab8f33",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-test_logical_compare_01-5db0e2e.stderr",
    "stderr_hash": "7b14d4f4d93adec3b96c94b6c769b68f504cfba4854d75e6e2dd7c3d",
    "returncode": 134
}