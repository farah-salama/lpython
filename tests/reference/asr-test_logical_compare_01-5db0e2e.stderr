LCOMPILERS_ASSERT failed: $DIR/src/lpython/semantics/python_ast_to_asr.cpp
function visit_BoolOp(), line number 3329 at 
ASRUtils::check_equal_type(ASRUtils::expr_type(lhs), ASRUtils::expr_type(rhs))
Aborted
